### DISCUSSION TITLE
Getting error while trying to run the local version to generate MacOS version of the pgml extension

### DISCUSSION THREAD

**Discussion by jjwmenting** [postgresml/postgresml#1547] [Q&A] [answered]
This is one of the errors i am getting and i am not sure how to fix this. I added the linfa deps directly from the linga repository.
I am new to Rust btw, mainly wanna use the extension locally on postgres instance without docker on MacOS and for Ubuntu server. If anybody could help me out that would be great.

`error[E0277]: the trait bound `argmin_param::ArgminParam<F, Dim<[usize; 1]>>: serde::ser::Serialize` is not satisfied
   --> deps/linfa/algorithms/linfa-logistic/src/lib.rs:830:19`

**Comment by levkk** [comment, marked as answer]
We include linfa as a Git submodule. Just do:

```bash
git submodule update --init --recursive
```

**Reply by jjwmenting** [reply]
Thank you for the quick response, it does build with the correct linfa version i guess.
